README.md

```diff
@@ -143,24 +143,29 @@ For a complete step-by-step tutorial, see [Getting Started](https://go.filecoin.
 # Remove any existing symlink to a repo directory
 rm ~/.filecoin
 
-# Initialize a new repository, downloading a genesis file and setting network parameters for the "interop" developer network
-./go-filecoin init --genesisfile=https://gateway.ipfs.io/ipfs/QmZ7ENWU2TYBuRfdXRjoEFvF6mNpYzyALPHexcgnonUeoK/interop.car --devnet-interop
+# Initialize a new repository, downloading a genesis file and setting network parameters (in this case, for the Testnet network)
+go-filecoin init --genesisfile=https://ipfs.io/ipfs/QmXZQeezX1x8uRQX9EUaYxnyivUpTfJqQTvszk3c8SnFPN/testnet.car --network=testnet
```

Comment:
is it --network=testnet2?

Reply:
its currently set to testnet.  I can change if deemed critical.  Since this is the first testnet phase gfc is active in it seems ok to me.

Reply:
No worries, I think we just wanted to make sure it was accurate.